docs/package.json

```diff
@@ -14,8 +14,8 @@
     "clean": "find docs/sources -type f -not -name '.keep' -print0 | xargs -0  -I {} rm {}"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.56",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.56",
+    "@docusaurus/core": "^2.0.0-alpha.58",
+    "@docusaurus/preset-classic": "^2.0.0-alpha.58",
```

Comment:
can you run `make install-docs`? that will update the `yarn.lock` file :)

Reply:
Oops, thought i did that. will do, thanks!